notes/stories/film/film-scifi-2010s.st.txt

```diff
@@ -3762,6 +3762,55 @@ father and son [Tony had a conversation with the 1970s version of his father whe
 sacrifice to prevent mass death [Hawkeye knowingly gave his life to retrieve one of the power stones, which the Avenger's needed to unto Thanos' murder of half the people on earth.]
 
 
+movie: Godzilla: King of the Monsters (2019)
+============================================
+
+:: Title
+Godzilla: King of the Monsters
+
+:: Description
+Godzilla: King of the Monsters is a 2019 American monster film directed and
+co-written by Michael Dougherty. A sequel to Godzilla (2014), it is the 35th
+film in the Godzilla franchise, the third film in Legendary's MonsterVerse,
+and the third Godzilla film to be completely produced by a Hollywood studio.
+
+Synopsis: Humans must rely on Godzilla and Mothra to defeat King Ghidorah, a
+terrible three-headed dragon-like monster that has awakened Rodan and other
+Titans to destroy the world.
+
+:: Date
+2019-05-13
+
+:: References
+https://en.wikipedia.org/wiki/Godzilla:_King_of_the_Monsters_(2019_film)
+
+:: Ratings
+1 <paul>
+
+:: Major Themes
+giant monster [The three-headed dragon-like monster King Ghidorah and a variety of other giant monsters under its control ravaged the cities of the world. In the end, the titular giant monster Godzilla came to the rescue of humanity.]
+human vs. monster [A variety of giant monsters (e.g. the three-headed dragon-like monster King Ghidorah, the giant pteranodon-like monster Rodan, and a host of other "Titans") were going around ravaging cities all around the world.]
+monster apocalypse [The three-headed dragon-like monster King Ghidorah awakened Rodan and other Titans to destroy the world.]
+the military professions [The G-Team, a special military forces group, was doing what it could to battle the various rampaging Titans.]
```

Comment:
Just thinking we might not want to use this theme for the special forces kind of action heroes you see in every damn action movie - not that I know what the situation was here, or where the line should be drawn.